lib/braintree/http.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
 let http = require('http');
+let Agent = require('http').Agent;
```

Comment:
Rather than creating `let _Agent` further down, let's just specify this as `HttpAgent`.

```suggestion
let HttpAgent = require('http').Agent;
```